src/portfolio.js

```diff
@@ -1,12 +1,13 @@
-﻿/* Change this file to get your personal Portfolio */
+﻿import emoji from "react-easy-emoji";
```

Comment:
I think you should move the import statement below the comment because the comment gives explanation for what the file does and should be at the top.